app/blog/2018-02-26-3.0-preview-paths-and-names.md

```diff
@@ -0,0 +1,159 @@
+
+The Node.js community has gotten used to this flexibility, and forcing all existing packages to adopt path-based flat installations is a hard sell.  
+
+
+## So what now?
+
+The current preview release (3.0-pre.10) still uses paths, so for now, keep doing what you're doing.
+
+Starting with the next preview release (3.0-pre.11), we're going to publish packages that use bare module specifiers to import from other packages.
+
+For example, where previous previews required a path like this:
+
+
+```bash
+../@polymer/polymer/polymer-element.js
+```
+
+
+Or:
+
+
+```bash
+/node_modules/@polymer/polymer/polymer-element.js
+```
+
+
+With the next release, you can refer to the Polymer package by name (`@polymer/polymer`), without regard to where it is installed:
+
+
+```bash
+@polymer/polymer/polymer-element.js
+```
+
+
+That is:
+
+
+
+*   _The name of the package_ (`@polymer/polymer`) replaces the _path to the package folder_. For   
+    scoped packages like the Polymer packages, the scope is part of the package name.
+*   For packages with a main module (identified in the `main` section of `package.json`), you can 
+    just use the package name instead of specifying the file path. For example, instead of 
+    `'lodash/index.js'` for the lodash package, you can just import `'lodash'`.
+
+This change does mean that you'll require some kind of build or transform step to run your Polymer code in the browser. The Polymer CLI tools will do this for you on the fly when you run `polymer serve` or WCT, and the Polymer CLI will rewrite names to paths at build time. 
+
+If you have a working project with paths, it should continue to work with the next release. (However, you'll still need to use a transform or build tool.)
+
+You're still free to author with paths if you like, but we strongly recommend using bare module specifiers everywhere—especially for reusable elements—because they're compatible with a wider range of tools.
+
+Here are the specific tooling changes we're making:
+
+
+
+*   Modulizer already has support for converting paths to bare specifiers when rewriting HTML 
+    imports to ES modules.
+*   We're adding support for bare specifiers to the Polymer CLI and related tools, as well as 
+    other projects like `prpl-node-server`.
+*   You can already preview code using bare module specifiers using `polymer serve`: 
+    
+    ```bash
+    polymer serve --module-resolution=node
+    ```
+
+Our support for bare specifiers is independent of our other build steps, like bundling. This step simply resolves specifiers and rewrites them as paths—`polyserve` will still serve modules, and `polymer-build` will output modules for browsers that support them. Bundling isn't required. 
+
+
+### Renaming Polymer.Element
```

Comment:
This section seems out of place since it's not related to the path->name transition.  How about we nix the section and instead just talk about this in the actual release blog post and/or notes?